File: backend/ms_sms/src/main/java/wevioo/tn/ms_sms/dtos/request/SendWhatsAppMsg.java
package wevioo.tn.ms_sms.dtos.request;

import lombok.Data;
import wevioo.tn.ms_sms.dtos.response.WhatsAppTemplateResponse;

import java.util.List;

@Data
public class SendWhatsAppMsg {
    private WhatsAppTemplateResponse whatsAppTemplateResponse;
    private List<String> numbers;
    private List<String> placeholders;
}

File: frontend/src/models/sms/SendWhatsAppMsg.ts
import { WhatsAppTemplateResponse } from "./WhatsAppTemplateResponse";

export interface SendWhatsAppMsg{
    whatsAppTemplateResponse:WhatsAppTemplateResponse;
    numbers:string[];
    placeholders:string[];
}

export interface SendWhatsappSeparately{
    number:string
    placeholders:string[];
}

export interface SendIndivWhatsapp{
    whatsAppTemplateResponse:WhatsAppTemplateResponse|null;
    sendSeparatelyList:SendWhatsappSeparately[];
}